src/System.Management.Automation/DscSupport/CimDSCParser.cs

```diff
@@ -667,7 +667,18 @@ public static void Initialize(Collection<Exception> errors, List<string> moduleP
             }
             else
             {
-                var systemResourceRoot = Path.Combine(Platform.GetFolderPath(Environment.SpecialFolder.System), "Configuration");
+                // DSC SxS scenario
+                var psAssembly = Assembly.GetEntryAssembly();
+                var configSystemPath = Path.GetDirectoryName(psAssembly.Location);
+                var systemResourceRoot = Path.Combine(configSystemPath, "Configuration");
+                var inboxModulePath = "Modules\\PSDesiredStateConfiguration";
```

Comment:
Will this module ever be published independently on PSGallery as this hard codes the path to the module.  Might be better to call GetModuleCommand to get the latest version on the system.

Reply:
@nitinbps @Indhukrishna

Reply:
Sorry for the delayed response. I don't think it's necessary to use the GetModuleCommand in this case since this change only fixes compilation in our DSC package and we'll always be getting the module from this location.